### PR TITLE
[5.1] docs(charts): update trendline articles to include the new types

### PR DESCRIPTION
**Pull request by xristianstefanov**
Related to: https://github.com/telerik/blazor/issues/7966.

### CODE REVIEW

File: components/chart/trendlines.md [hunk 1]
```diff
@@ -16,11 +16,18 @@ Trendline series use the data from the main series. When the main series employs
 
 ## Trendline Types
 
-The Chart supports two fundamental types of trendlines - the `Linear Trendline` and the `Moving Average Trendline`.
+The Chart supports the following fundamental types of trendlines:
+
```
Comment (ikoevska) [inline components/chart/trendlines.md:20]: Drop the monospace formatting in the list.

File: components/chart/trendlines.md [hunk 2]
```diff
@@ -16,11 +16,18 @@ Trendline series use the data from the main series. When the main series employs
...
+* [`Moving Average Trendline`](#moving-average-trendline)
+* [`Exponential Trendline`](#exponential-trendline)
+* [`Logarithmic Trendline`](#logarithmic-trendline)
+* [`Power Trendline`](#power-trendline)
+* [`Polynomial Trendline`](#polynomial-trendline)
 
 ### Linear Trendline
 
-The Linear Trendline serves the purpose of indicating whether a specific quantity is on the rise or decline over time.
+The `Linear Trendline` serves the purpose of indicating whether a specific quantity is on the rise or decline over time.
```
Comment (ikoevska) [inline components/chart/trendlines.md:30]:
```suggestion
Use the Linear Trendline to visualize the rise or decline of a specific quantity over time.
```

File: components/chart/trendlines.md [hunk 3]
```diff
@@ -174,7 +181,7 @@ The Linear Trendline serves the purpose of indicating whether a specific quantit
 
 ### Moving Average Trendline
 
-The Moving Average Trendline smoothens out data fluctuations by computing an average of all data points over a specified period. By default, this period is set to `2` chart intervals.
+The `Moving Average Trendline` smoothens out data fluctuations by computing an average of all data points over a specified period. By default, this period is set to `2` chart intervals.
```
Comment (ikoevska) [inline components/chart/trendlines.md:184]:
```suggestion
Use the Moving Average Trendline to smoothen out data fluctuations. This trendline computes and visualizes an average of all data points over a specified period. By default, this period is set to two chart intervals.
```

File: components/chart/trendlines.md [hunk 4]
```diff
@@ -313,12 +320,758 @@ The Moving Average Trendline smoothens out data fluctuations by computing an ave
 }
 `````
 
+### Exponential Trendline
+
+The `Exponential Trendline` shows data with rapidly accelerating growth or decay over time. It's used to emphasize trends that follow an exponential pattern, and it requires positive `Y` values.
```
Comment (ikoevska) [inline components/chart/trendlines.md:325]:
```suggestion
Use the Exponential Trendline to visualize data with rapidly accelerating growth or decay over time. This trendline helps you to emphasize trends that follow an exponential pattern, and it requires positive `Y` values.
```

File: components/chart/trendlines.md [hunk 5]
```diff
@@ -313,12 +320,758 @@ The Moving Average Trendline smoothens out data fluctuations by computing an ave
...
+        public int NumericPeriod { get; set; }
+        public DateTime Date { get; set; }
+        public decimal Price { get; set; }
+    }
+}
+`````
+
+### Logarithmic Trendline
+
+The `Logarithmic Trendline` is applied to data that displays rapid initial growth that slows over time or vice versa. It emphasizes trends with a logarithmic pattern, and it requires positive `X` values.
```
Comment (ikoevska) [inline components/chart/trendlines.md:511]:
```suggestion
Use the Logarithmic Trendline to visualize data with rapid initial growth that slows over time or vice versa. This trendline emphasizes trends with a logarithmic pattern, and it requires positive `X` values.
```

File: components/chart/trendlines.md [hunk 6]
```diff
@@ -313,12 +320,758 @@ The Moving Average Trendline smoothens out data fluctuations by computing an ave
...
+        public int NumericPeriod { get; set; }
+        public DateTime Date { get; set; }
+        public decimal Price { get; set; }
+    }
+}
+`````
+
+### Power Trendline
+
+The `Power Trendline` is used for data that follows a power-law relationship, indicating that one variable's change is proportional to a power of another variable. It helps highlight trends where the rate of change isn't constant. The `Y` and `X` require positive values.
```
Comment (ikoevska) [inline components/chart/trendlines.md:697]:
```suggestion
Use the Power Trendline to visualize data that follows a power-law relationship, indicating that one variable's change is proportional to a power of another variable. This trendline helps you to highlight trends where the rate of change isn't constant. It requires positive `Y` and `X` values.
```

File: components/chart/trendlines.md [hunk 7]
```diff
@@ -313,12 +320,758 @@ The Moving Average Trendline smoothens out data fluctuations by computing an ave
...
+        public int NumericPeriod { get; set; }
+        public DateTime Date { get; set; }
+        public decimal Price { get; set; }
+    }
+}
+`````
+
+### Polynomial Trendline
+
+The `Polynomial Trendline` is applied to data with complex patterns that cannot be captured by simpler trendlines. It fits a polynomial equation to the data points, allowing for a more flexible representation of trends with multiple turning points.
```
Comment (ikoevska) [inline components/chart/trendlines.md:884]:
```suggestion
Use the Polynomial Trendline to visualize complex data patterns not fitting the other trendlines. This trendline fits a polynomial equation to the data points, allowing for a more flexible representation of trends with multiple turning points.
```

**Review by ikoevska — COMMENTED** [review]
General notes from the review:
* You don't really need the monospace formatting in the bulleted list at the beginning of the page. In fact, you can drop it from all instances. It makes sense to use monospace formatting if you're using the correct spelling used in the code.
* I took the liberty of suggesting a "Use X to achieve Y" template for the descriptions of the charts. It's more action-oriented and provides clear goals at a glance. Feel free to rework the suggestions as needed, if I messed up the accuracy.
* Personally, I wonder if Supported Series should be moved to the beginning of the page. So that if you're working with something that's not supported, you could safely skip the page.
* This one seems out of the scope of these changes, but I would have loved to see a more detailed explanation about how to configure/implement trendlines (I know it's obvious in the code, but ideally, there would be a section/a paragraph/one line saying where and when in your code it's appropriate to implement it, possibly, a general template or any extra info needed to make it work, what are some best practices for Blazor trendlines (if any), tips and tricks (or links to them, possibly links to applicable KBs), styling.

File: components/chart/trendlines.md [hunk 8]
```diff
@@ -313,12 +320,758 @@ The Moving Average Trendline smoothens out data fluctuations by computing an ave
...
+    {
+        public string Period { get; set; }
+        public int NumericPeriod { get; set; }
+        public DateTime Date { get; set; }
+        public decimal Price { get; set; }
+    }
+}
+`````
+
 ## Chart Trendlines Parameters
```
Comment (dimodi) [inline components/chart/trendlines.md:1068]: I believe it makes sense to move the theory above the examples.

**Review by ikoevska — APPROVED** [review]
🚀

**Pull request merged**